vector/src/main/java/im/vector/adapters/VectorMessagesAdapterHelper.java

```diff
@@ -171,10 +171,12 @@ public void setSenderValue(View convertView, MessageRow row, boolean isMergedVie
         if (null != senderTextView) {
             Event event = row.getEvent();
 
+            // Hide the group flair by default
+            groupFlairView.setVisibility(View.GONE);
+            groupFlairView.setTag(null);
+
```

Comment:
you might have flair icon flickerings on low end devices...

Reply:
@ylecollen I did some tests without observing flickering. I saw some delay in the flair display sometimes, but it was already the case before my changes. So I will keep my changes FTM, thank you to have warned me.